docs/content.zh/docs/deployment/resource-providers/native_kubernetes.md

```diff
@@ -324,4 +324,235 @@ $ kubectl create clusterrolebinding flink-role-binding-flink --clusterrole=edit
 
 Please refer to the official Kubernetes documentation on [RBAC Authorization](https://kubernetes.io/docs/reference/access-authn-authz/rbac/) for more information.
 
+### Pod Template
+
+Flink allows users to define the JobManager and TaskManager pods via template files, with the advanced features (e.g. volumes, init containers, sidecar containers, etc.)
```

Comment:
Flink allows users to define the JobManager and TaskManager pods via template files. This allows to support advanced features